### ISSUE TITLE
Suggestion: improve mouse settings GUI (sensitivity, acceleration)

### ISSUE THREAD

**Issue by rjurga** [budgie-desktop/budgie-desktop#1080]
Hello,

Currently changing mouse settings in Solus is cumbersome. Some settings are accessible through two different applications, and some settings like sensitivity are available nowhere.
- Mouse & Touchpad: only allows changing the primary button of the mouse.
- Tweak Tool: it has some more options, including acceleration profile.

It took me a while to find the setting to disable mouse acceleration because I intuitively expected to find it in the Mouse & Touchpad menu, not in Tweak Tool. The user experience for configuring the mouse on Solus would be improved if the most important settings were all available in one configuration menu.

Also most people probably know why sensitivity configuration matter, but less so acceleration. So a brief justification is that many gamers playing first person shooters like to disable it. It's not always configurable through the game settings themselves so having the option to easily disable it OS-wide helps. Some gamers even like to enable more complex acceleration profiles, such as with the Povhat's mouse acceleration driver in Windows, but I think asking for something like this on Solus is a bit too much given its niche usage, I'm just mentioning it to illustrate its importance for gaming.

To sum up, the issues are:
- Two different menus to configure mouse
- No menu offer sensitivity configuration

The initial user experience would be smoother with a single menu offering the most useful mouse configurations.

I hope it is the right place to make such a suggestion, and thanks for all the work on such an awesome distro.

Bye,
Radek

**Comment by awareofdistractions** [comment]
More advanced settings for keyboard (e.g repeat delay and rate) would be also appreciated.
Right now I am using `xset r rate DELAY RATE`

**Comment by rjurga** [comment]
Update concerning Solus 3.

Solus 3 comes without Tweak Tool. With a fresh install, the only available mouse configuration tool is Mouse & Touchpad. This means that currently, Solus doesn't ship with any meaningful mouse configuration tool that would allow to change the sensitivity and the acceleration settings.

**Comment by Frampis** [comment]
I really want to enable true 1:1 mouse for gaming. In Windows it is the setting that results from sensitivity 6/11 and "enhance pointer precision" off.

![Screenshot from Windows](https://user-images.githubusercontent.com/8354539/33358179-9f0a9862-d4cf-11e7-8ebc-c9c9f023f14c.png)

**Comment by KWottrich** [comment]
I would also like to see a way to get to cursor behavior equivalent to what @Frampis said is possible on Windows.

**Comment by atillabyte** [comment]
I've noticed that, unfortunately, every distro I've tried (on multiple laptops) never had a cursor that felt the same way as it does on Windows. It's difficult to describe, but it's certainly less precise and sometimes erratic and jumpy. I usually have to migrate back to Windows because it's too much of an inconvenience and hassle for a desktop operating system.

I don't really know the technicalities and what Microsoft had implemented to achieve their mouse precision, or whether it's simply an issue with drivers. My initial impression is that people more or less concede and/or get used to the difference. I'd like to know if there's anyone who _does_ have the exact same cursor functionality with Linux as they would with Windows, when using a track pad.

**Comment by 3vilgenius** [comment]
Hello, I also want to see a mouse settings gui or at SOMETHING that allows me turn off mouse acceleration. I don't think I will be switching until that happens.
